src/messaging/ExchangeContext.h

```diff
@@ -33,6 +33,9 @@
 #include <transport/SecureSessionMgr.h>
 
 namespace chip {
+
+constexpr uint16_t kMsgCounterChallengeSize = 8; // The size of the message counter synchronization request message.
```

Comment:
Should we move this to `MessageCounterSync.h`?

Reply:
This will cause mutual including during compiling. 

The mChallenge should not be declared in ExchangeContext.h, it will be moved to MessageCounterSync.h after:
// [TODO: #4711]: this field need to be moved to appState object which implement 'exchange-specific' contextual actions with a delegate pattern.